app/cu_editor.py

```diff
@@ -32,6 +32,8 @@ def initCommandSet(editText, textBuffer):
 
     CTRL_H: textBuffer.backspace,
 
+    CTRL_L: textBuffer.selectCurrentLine,
```

Comment:
Is the intent to have ctrl+L operate everywhere, including the prompt UIs at the bottom of the screen (like Find and Goto line)? If so this is the spot for this line.

If not (like if the intent is for ctrl+L line selection to be in the main edit window) then please move this down to the CuaPlusEdit class around line 312.